### PR TITLE
Feature/sep6 26 create login page

### PR DESCRIPTION
**Pull request by Tabita-Roxana**

### CODE REVIEW

File: frontend/src/Pages/sign-up.tsx [hunk 1]
```diff
@@ -1,28 +1,87 @@
...
+    gender: string,
+    role:string
+};
 
 const SignUp = () => {
 
   const { classes } = useStyles();
+  const navigate = useNavigate();
+
+  const { register, setValue, handleSubmit, formState: { errors } } = useForm<Account>();
```
Comment (GeorgianaIon) [inline frontend/src/Pages/sign-up.tsx:26]: You can delete setValue if you don't use it

File: frontend/src/Pages/sign-up.tsx [hunk 2]
```diff
@@ -1,28 +1,87 @@
...
+  const onSubmit = handleSubmit(data => handleSignup(data));
+
+
+  const handleSignup = async (data: Account) => {
+  //e.preventDefault();
```
Comment (GeorgianaIon) [inline frontend/src/Pages/sign-up.tsx:31]: remove this

File: frontend/src/Pages/login.tsx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+};
+
+const useStyles = makeStyles()(() => ({
+  login_form:{
+    display:'grid',
+    justifyContent: 'center',
+    alignItems:'center'
+  },
+  
+  login_button:{
```
Comment (GeorgianaIon) [inline frontend/src/Pages/login.tsx:78]: You don't need all the design for the button since it is defined in theme

**Comment by Tabita-Roxana** [comment]
pair programming with Georgiana for using react-hook

**Pull request merged**